test/e2e/registry_mirror_test.go

```diff
@@ -12,9 +12,9 @@ import (
 
 func runRegistryMirrorConfigFlow(test *framework.ClusterE2ETest) {
 	test.GenerateClusterConfig()
-	test.ImportImages()
+	// test.ImportImages()
 	test.CreateCluster()
-	test.ImportImages()
+	// test.ImportImages()
```

Comment:
can we just remove these lines?
Or if they serve some kind of documentation purpose then comment why they are there

Reply:
those lines are needed, forgot to uncomment 😅